numpy/core/numeric.py

```diff
@@ -1772,30 +1772,34 @@ def _frombuffer(buf, dtype, shape, order):
 
 
 @set_module('numpy')
-def isscalar(num):
+def isscalar(element):
```

Comment:
Are we ok with renaming this argument? I think the chance of it breaking anyone is ~0.

Reply:
it seems harmless